source/features/update-pr-from-base-branch.tsx

```diff
@@ -104,7 +98,9 @@ async function init(): Promise<void | false> {
 	// - Draft PR (show the button anyway)
```

Comment:
This shouldn't exist anymore. Maybe the condition is `hasConflicts || !canMerge`